app/controllers/repository_rows_controller.rb

```diff
@@ -365,8 +370,15 @@ def load_repository_row_print
   end
 
   def load_repository_or_snapshot
-    @repository = Repository.accessible_by_teams(current_team).find_by(id: @repository_row&.first&.repository_id)
-    @repository ||= RepositorySnapshot.find_by(id: @repository_row&.first&.repository_id)
+    if @repository_row
```

Comment:
Doesn't look right, you already have repository_row, why do you search for repository?